contracts/HeapOrdering.sol

```diff
@@ -13,6 +13,10 @@ library HeapOrdering {
         mapping(address => uint256) ranks; // A mapping from an address to a rank in accounts. Beware: ranks are shifted by one compared to indexes, so the first rank is 1 and not 0.
     }
 
+    /// CONSTANTS ///
+
+    uint256 public constant ROOT = 1;
```

Comment:
I don't think we want to make it public: it is an implementation detail

Reply:
Ok, changing it

Reply:
we don't really care no? We can keep it public no? We're building ob public blockchains 🤓

Reply:
Yes sure, it's a really small change, but it looks a bit ugly if that is the only thing public in this contract, don't you think ?

Reply:
Yes I agree